packages/updatePrescriptionStatus/src/updatePrescriptionStatus.ts

```diff
@@ -284,6 +286,33 @@ function response(statusCode: number, responseEntries: Array<BundleEntry>) {
   }
 }
 
+async function logTransitions(dataItems: Array<DataItem>, logger: Logger): Promise<void> {
+  for (const dataItem of dataItems) {
+    try {
+      const previousItem = await getPreviousItem(dataItem)
+      if (previousItem) {
+        const newDate = new Date(dataItem.LastModified)
+        const previousDate = new Date(previousItem.LastModified)
+        logger.info("Transitioning item status.", {
+          prescriptionID: dataItem.PrescriptionID,
+          lineItemID: dataItem.LineItemID,
+          nhsNumber: dataItem.PatientNHSNumber,
+          pharmacyODSCode: dataItem.PharmacyODSCode,
+          applicationName: dataItem.ApplicationName,
+          when: dataItem.LastModified,
+          interval: (newDate.valueOf() - previousDate.valueOf()) / 1000,
+          newStatus: dataItem.Status,
+          previousStatus: previousItem.Status,
+          newTerminalStatus: dataItem.TerminalStatus,
+          previousTerminalStatus: previousItem.TerminalStatus
+        })
+      }
```

Comment:
Is it worth tracking transitions from 'no state' (ie. new items)?
Would we be interested in the distribution of 'entry states', or the first state psu receives an update for?

Reply:
We already have logs of all requests and it's primarily that interval between states that was the missing data we needed so no need I would say.